README.md

```diff
@@ -0,0 +1,74 @@
+# mattermost-github-integration
+
+This role sets up a mattermost github integration service which posts changes of an organization or repository to a mattermost channel.
+
+
+## Requirements
+
+This role works standalone
+
+
+## Role Variables
+
+| Name                                           |         Required         | Default                              | Description                                                         |
```

Comment:
Also, why don't you merge Required and Default?

Reply:
because i like it this way